### ISSUE TITLE
Order and SKU are not exported in stripe v11.7.0

### ISSUE THREAD

**Issue by charliemr99** [kgajera/stripe-event-types#4]
```sh
[12:28:04 PM] File change detected. Starting incremental compilation...

node_modules/stripe-event-types/index.d.ts:519:46 - error TS2694: Namespace '"stripe".Stripe' has no exported member 'Order'.

519         data: DiscriminatedEvent.Data<Stripe.Order>;
                                                 ~~~~~

node_modules/stripe-event-types/index.d.ts:642:46 - error TS2694: Namespace '"stripe".Stripe' has no exported member 'Sku'.

642         data: DiscriminatedEvent.Data<Stripe.Sku>;
                                                 ~~~

[12:28:15 PM] Found 2 errors. Watching for file changes.
```

**Comment by kgajera** [comment]
Thanks for reporting! Will get a fix out for this soon.

**Comment by kgajera** [comment]
Fixed in [v2.0.0](https://github.com/kgajera/stripe-event-types/releases/tag/v2.0.0).
